Added events to the extender api (#419)
zowe/cics-for-zowe-client: packages/vsce/CHANGELOG.md

All notable changes to the "cics-extension-for-zowe" extension will be documented in this file.

## Recent Changes

> - Enhancement: Added and event to the Resource Inspector command so the subscribers can be notified os this. [#419](https://github.com/zowe/cics-for-zowe-client/pull/419)

Reviewer: ```suggestion
- Enhancement: Added an event to the Resource Inspector command so the subscribers can be notified of this. [#419](https://github.com/zowe/cics-for-zowe-client/pull/419)
```
